### ISSUE TITLE
Please provide pdf version

### ISSUE THREAD

**Issue by sorinmuchi** [ResidentMario/ml-project-scoping-talk#1]
Running arbitrary js code sounds pretty dangerous from a security perspective. Please provide a pdf version of the slides. 

https://revealjs.com/pdf-export/

**Comment by ResidentMario** [comment]
Fair point. Unfortunately something about how I formatted the images in this slide deck causes them to slide off the screen when exported to PDF. Will need to figure out what is going on there and make that change first.
